components/form.css

```diff
@@ -1,3 +1,23 @@
-# {
-  background-color: red;
+.headlineForm {
+  background-color: var(--background-color);
+  font-size: 2rem;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  padding-inline: 30px;
+  font-size: 1.5rem;
+}
+
+label {
+  padding: 30px 0 10px 0;
+}
+
+textarea {
+  height: 100px;
+}
+
+input {
+  height: 50px;
```

Comment:
for good practice please use only classes. This is a pattern that we will use the next months, so getting used to it is very helpful.